package.json

```diff
@@ -18,24 +18,28 @@
     "url": "https://github.com/icelab/formalist-compose/issues"
   },
   "scripts": {
-    "compile": "./node_modules/.bin/babel --presets ./node_modules/babel-preset-es2015 src --out-dir lib",
+    "compile": "babel --presets babel-preset-es2015 src --out-dir lib",
     "prepublish": "npm run compile",
-    "test": "./node_modules/.bin/babel-node --presets ./node_modules/babel-preset-es2015 test | faucet",
-    "lint": "./node_modules/.bin/eslint 'src/*.js' 'src/**/*.js' 'test/*.js' 'test/**/*.js'",
+    "test": "babel-node --presets babel-preset-es2015 test | faucet",
```

Comment:
To my knowledge it should look locally first. So as long as we specify all the modules we use in the `package.son` we should be sweet. Perhaps give it a run and see if you have issues - personally I haven't come across anything like that.